analysis/demonhunterhavoc/src/CONFIG.js

```diff
@@ -1,14 +1,15 @@
+import { ToppleTheNun } from 'CONTRIBUTORS';
 import Expansion from 'game/Expansion';
 import SPECS from 'game/SPECS';
 
 import CHANGELOG from './CHANGELOG';
 
 export default {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
-  contributors: [],
+  contributors: [ToppleTheNun],
   expansion: Expansion.Shadowlands,
   // The WoW client patch this spec was last updated.
-  patchCompatibility: null,
+  patchCompatibility: '9.2.5',
```

Comment:
I haven't reviewed the content from this spec recently - are things up to date across the board? If you're confident we can put this through as well, but I don't want it to be automatic.

Reply:
I'll remove this check for the moment and then do a quick run through after my TS PR to make sure everything's good to go for 9.2.5/9.2.7.